### ISSUE TITLE
Update guideline bullets in the "Contribute" section in README file.

### ISSUE THREAD

**Issue by lugenx** [lugenx/ecohabit#120]
remove this line:
- If there's no existing Issue for which PR need to be raised, first create an Issue in the project and then create the PR.

add these lines:

- If there's no existing Issue for which PR needs to be raised, first create an Issue in the project and then wait for it to get assigned.
- Before starting work on a new pull request, make sure you have been assigned to the related issue. To get assigned, please comment on the issue.

**Comment by amiranga** [comment]
Hi @lugenx, could I work on this?

**Comment by lugenx** [comment]
yes please, go ahead, thank you.
